fridays-with-sasha-6pm-to-8pm/README.md

```diff
@@ -14,5 +14,6 @@ Add your name to request a ticket for admittance
 
 Add your name to request a ticket for admittance
 
- -
+ 1. Nick Lavery
```

Comment:
Nice job adding the comment.Noe that since this is an md file vs a js file, no forward slashes are needed.

Reply:
Understood! I'll review the MD formatting, definitely something I could use more practice with

Reply:
No worries at all. That is why we have this in a repo.  More practice and practice makes permanent.

A good bedtime md read:
https://guides.github.com/features/mastering-markdown/